tests/probe/t15-specimen.sh

```
@@ -0,0 +1,66 @@
+
+# ---------------------------------------------------------------- region A ---
+
+sum_sizes() {
+  DIR=$1
+  total=0
+  cd $DIR
+  for f in `ls $DIR`; do
+    sz=`cat $DIR/$f | wc -c`
+    total=`expr $total + $sz`
+  done
+  echo $total > /tmp/sizes.txt
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)